js/whosehouse.js

```diff
@@ -57,6 +59,16 @@ function displayWhoseHouseWinner() {
   baselineRotation = Math.floor(finalRotation / 360) * 360;
   console.log('houses: ' + wh.houses.join());
   console.log('winner: ' + winnerHouse);
+
+  // Bold the winner when done rotating
+  $('#wheel').bind(transition_end_events,
+      (function(winnerHouse) {
+        return function() {
+          $('#wheel-' + winnerHouse).css('font-weight', 'Bold');
```

Comment:
nit: I don't think `Bold` needs to be capitalized

Reply:
okay, and `Normal`